### DISCUSSION TITLE
Question about copyright licensing using screenshots from backtesting.py charts

### DISCUSSION THREAD

**Discussion by botradingblog1** [kernc/backtesting.py#690] [General]
Hi,

I wanted to find out if I can use screenshots of the backtesting.py dynamically generated charts on my personal financial blog that has a monthly subscription fee?
I am mentioning the chart is generated with backtesting.py.

Please let me know.

**Comment by kernc** [comment]
Of course, as the [License section](https://kernc.github.io/backtesting.py/doc/backtesting/#license) somewhat succintly states, the strategies and the results, including the Bokeh-based charts, are wholly yours to do with as you please.

An attribution link sure is appreciated, though. 😊

**Reply by botradingblog1** [reply]
Thanks for the clarification! :)
